pennylane/workflow/qnode.py

```diff
@@ -479,6 +482,18 @@ def __init__(
                 gradient_kwargs,
             )
 
+        if expansion_strategy is not None:
+            warnings.warn(
+                "The 'expansion_strategy' attribute is deprecated and will be removed  "
+                "in version 0.39. For full control over the stage to which the tape is "
+                "constructed, use the 'pennylane.workflow.construct_batch' function.",

```

Comment:
Is this the right advice :thinking: As far as I know, this value is used as a fallback for `draw`, `specs`, `draw_mpl` when `expansion_strategy` isn't specified there. What else is the QNode's `expansion_strategy` used for?

Reply:
It is also used in the `construct` function of the `QNode` [here](https://github.com/PennyLaneAI/pennylane/blob/f1b5e31110a58c38712f8f3251d6bb6a76ea5cba/pennylane/workflow/qnode.py#L948), and therefore it is having its different deprecation message. For the functions you mentioned, it's much more straightforward.